src/modules/paymentProcessor/PP_Streaming_v1.sol

```diff
@@ -948,23 +957,48 @@ contract PP_Streaming_v1 is Module_v1, IPP_Streaming_v1 {
         );
     }
 
-    function _getStreamingDetails(bytes32 flags, bytes32[] memory data)
-        internal
-        view
-        returns (uint start, uint cliff, uint end)
-    {
-        uint dataIdx = 0;
+    function _getStreamingDetails(
+        bytes32 order_flags,
+        bytes32[] memory order_data
```

Comment:
Why are the parameters using snake case and not camel case?
Also: the underscores at the end are missing as well.